projects/ngx-mask-lib/src/lib/mask.service.ts

```diff
@@ -420,7 +419,8 @@ export class MaskService extends MaskApplierService {
 
 		const separatorPrecision: number | null = this._retrieveSeparatorPrecision(this.maskExpression);
 		let separatorValue: string = this._retrieveSeparatorValue(result);
-		if (this.decimalMarker !== '.') {
+
+		if (this.decimalMarker !== '.' && !Array.isArray(this.decimalMarker)) {
 			separatorValue = separatorValue.replace(this.decimalMarker, '.');
```

Comment:
If you have an array of decimalMarker like ['.',','], the value will not be converted to decimal. For example 10,45 returns NaN in _checkPrecision